app/scripts/modules/amazon/src/loadBalancer/details/loadBalancerDetails.html

```diff
@@ -41,12 +41,12 @@ <h3 select-on-dbl-click>
           <ul class="dropdown-menu" uib-dropdown-menu role="menu">
             <li><a href ng-click="ctrl.editLoadBalancer()">Edit Load Balancer</a></li>
             <!--<li><a href ng-click="ctrl.cloneLoadBalancer()">Clone Load Balancer</a></li>-->
```

Comment:
If this has been commented out for a while, might as well remove it and remove the code in the controller...